test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/function/BiFunctionTest.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.javalib.util.function
+
+import java.util.function.{Function, BiFunction}
+
+import org.junit.Assert._
+import org.junit.Test
+
+class BiFunctionTest {
+  import BiFunctionTest._
+
+  @Test def create_and_apply(): Unit = {
+    assertEquals(3, addBiFunc(1, 2))
+  }
+
+  @Test def andThen(): Unit = {
+    assertEquals(4, addBiFunc.andThen[Int](incFunc)(1, 2))
```

Comment:
This type hint probably isn't required, I think I originally just had it in there because I forgot to import `ju.Function` and it was using the scala Function